vofs/vofs/tests/test_vofs.py

```diff
@@ -742,10 +740,8 @@ def mock_read(block_size):
 
         # Truncate a non-open file past the start of the file.
         testfs.cache.open.reset_mock()
-        with nested(patch('vofs.CadcCache.FileHandle.release'),
-                    patch('vofs.CadcCache.FileHandle.readData')) as mocks:
-            mockRelease = mocks[0]
-            mockReadData = mocks[1]
+        with patch('vofs.CadcCache.FileHandle.release') as mockRelease, \
+                    patch('vofs.CadcCache.FileHandle.readData') as mockReadData:
             mockRelease.wraps = origRelease  # TODO This doesn't really work,
```

Comment:
The test itself doesn't seem to be failing - it just leaking open files. That's because the original release and truncate in the FileHandler are not called for some reason. I've tried to track down the problem but it's not so obvious and I don't think it's worth the time and effort.